### ISSUE TITLE
Lambda Node.js runtime doesn't support v6.10 anymore

### ISSUE THREAD

**Issue by thoop** [jinty/prerender-cloudfront#12]
The Cloudformation template has:

Runtime: "nodejs6.10"

But as of June 4th, 2019 the currently supported Lambda runtimes for Node are:

nodejs8.10
nodejs10.x

https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

I don't think modifying the runtime would change anything but wanted to put that out there first.

**Comment by jinty** [comment]
Thanks for the note. I'll don't think there will be any issues with changing the runtime either. I'll try find the time to test with an updated runtime. Unless someone else beats me to it.

**Comment by benjaminlong** [comment]
Hello, I've tested your repo recently with my aws account.
Regarding nodejs, here the modification I needed to do:
https://github.com/benjaminlong/prerender-cloudfront/commit/76d8c55341090316317b7ac9f1574d93e88b929c

Then it worked like a charm. I just followed the README.
Note: I'm currently not using the new aws stack on production yet. I have to migrate my previous bucket to the new one. By the way, I was wondering if we can setup the new aws stack to point to an existing s3 bucket hosting the website ?!
 
All the best,
Thanks @jinty and all the contributors of the original project !
